testsuite/admin-client-tests/src/test/java/org/keycloak/client/testsuite/AbstractAdminClientTest.java

```diff
@@ -212,4 +212,12 @@ protected RealmResource testRealm() {
     public RealmResource testRealmResource() {
         return adminClient.realm("test");
     }
-}
\ No newline at end of file
+
+    protected String randomAlphanumericString(int count) {
```

Comment:
Any reason not to use `UUID.randomUUID()` here instead?

Reply:
This method is used for testing, previously it was `RandomStringUtils.random` using the `commons-lang` dependency obtained from quarkus. It is used to obtain some long string for attributes. I just created a helper in the abstract class to substitute it. The `randomUUID` is of a fixed length. We can do anything here, I preferred to not include more dependencies in the CI. But if you prefer something else just let me know.